### PR TITLE
HPCC-21829 Added HU Translations

### PR DESCRIPTION
**Pull request by kunalaswani**
Added Hungarian translations for 7.2.x

Signed-off-by: Aswani, Kunal <kunal.aswani@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [x] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by kunalaswani** [comment]
@AttilaVamos please review

File: esp/src/eclwatch/nls/hu/hpcc.js [hunk 1]
```diff
@@ -103,6 +103,7 @@ define(
     Compress: "Tömörít",
     Compressed: "Tömörített",
     CompressedFileSize: "Tömörített fájl méret",
+    ComputerUpTime: " A számítógép újraindítása óta eltelt idő ",
```
Comment (AttilaVamos) [inline esp/src/eclwatch/nls/hu/hpcc.js]: Extra leading and trailing space

File: esp/src/eclwatch/nls/hu/hpcc.js [hunk 2]
```diff
@@ -113,9 +114,12 @@ define(
     Contents: "Tartalom",
     ContentType: "Tartalom típusa",
     ContinueWorking: "Munka folytatása",
+    Copied: " Lemásolva ",
```
Comment (AttilaVamos) [inline esp/src/eclwatch/nls/hu/hpcc.js]: Extra leading/trailing spaces in some new lines

**Review by AttilaVamos — CHANGES_REQUESTED** [review]
@kunalaswani It seems right, but there are some extra leading/trailing spaces

**Comment by kunalaswani** [comment]
@AttilaVamos please review, spaces removed.

**Review by AttilaVamos — APPROVED** [review]
@kunalaswani It is good.

**Pull request merged**